doc/source/cookbook.rst

```diff
@@ -1362,11 +1421,11 @@ of the data values:
 
 
    def expand_grid(data_dict):
-      rows = itertools.product(*data_dict.values())
-      return pd.DataFrame.from_records(rows, columns=data_dict.keys())
+       rows = itertools.product(*data_dict.values())
+       return pd.DataFrame.from_records(rows, columns=data_dict.keys())
 
    df = expand_grid(
-      {'height': [60, 70],
-       'weight': [100, 140, 180],
-       'sex': ['Male', 'Female']})
+       {'height': [60, 70],
```

Comment:
Minor thing, but if you want to move this to the previous line, this is how in the previous examples is done.